qemu/tests/balloon_check.py

```diff
@@ -182,11 +182,11 @@ def get_memory_boundary(self, balloon_type=''):
         if self.params.get('os_type') == 'windows':
             logging.info("Get windows miminum balloon value:")
             self.vm.balloon(1)
-            balloon_timeout = self.params.get("balloon_timeout", 900)
-            self.wait_for_balloon_complete(balloon_timeout)
-            used_size = int(self.get_ballooned_memory() + self.ratio * self.ori_mem)
+            self.wait_for_balloon_complete(self.get_ballooned_memory, 0)
+            used_size = int(self.get_ballooned_memory())
             self.vm.balloon(max_size)
-            self.wait_for_balloon_complete(balloon_timeout)
+            #For windows flexible used memory,define a tolerance scope "5M" here
+            self.wait_for_balloon_complete(self.get_memory_status, 5)
```

Comment:
I have done  test on my environment for times, tolerance scope of "5M" is ok, but on larger memory environment, i'm not sure. So, the tolerance scope definition depends on your feedback, thanks.